java-auto-instrumetation/deploy/role.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  creationTimestamp: null
+  name: java-auto-instrumetation
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - services
+  - services/finalizers
+  - endpoints
+  - persistentvolumeclaims
+  - events
+  - configmaps
+  - secrets
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  - daemonsets
+  - replicasets
+  - statefulsets
+  verbs:
```

Comment:
I think this is a pretty long list of actions. I am wondering if we can limit to only those that are needed

Reply:
I've created https://jira.kumoroku.com/jira/browse/TRACING-705 for that